our-work/collaborations/crunch.yml

```diff
@@ -0,0 +1,15 @@
+title: The Crunch Group
+id: crunch-group
+description: The CRUNCH group is a research team of the Division of Applied Mathematics working at the interface of Computational Mathematics + Machine Learning + X, where X may be problems in biology, geophysics, soft matter, functional materials, physical chemistry, or fluid and solid mechanics. Their emphasis is on Scientific Machine Learning and Physics-Informed Neural Networks (PINNs) that the CRUNCH group pioneered. 
```

Comment:
At the end of the description add a sentence or two about CCV engagement's with the group. For instance in other collars we have said
>CCV's collaboration with BrainGate focused around the development of a database solution using TimeScaleDB for continuous at-home neuro-recording.